playwright-e2e/.env.sample

```diff
@@ -28,6 +28,9 @@ DSM_USER1_PASSWORD=
 # third user
 DSM_USER2_EMAIL=
 DSM_USER2_PASSWORD=
+# fourth user with very limited kit_shipping permissions
```

Comment:
Not to throw a wrench in the works here but we probably should have made the second user DSM_USER2, the third DSM_USER3, etc.

Reply:
will do